src/modules/peek/Peek.FilePreviewer/FilePreview.xaml.cs

```diff
@@ -41,17 +43,29 @@ public FilePreview()
 
         public IBitmapPreviewer? BitmapPreviewer => Previewer as IBitmapPreviewer;
 
+        public IBrowserPreview? BrowserPreviewer => Previewer as IBrowserPreview;
+
         public bool IsImageVisible => BitmapPreviewer != null;
 
-        public File File
+        /* TODO: need a better way to switch visibility according to the Preview.
+         * Could use Enum + Converter to switch according to the current preview. */
+        public bool IsBrowserVisible
```

Comment:
I think this could be reduced to `public bool IsBrowserVisible => BrowserPreviewer?.IsPreviewLoaded ?? false;`

Reply:
I don't think this will get updated when the IsPreviewLoaded changed. Maybe I am missing something but I don't see the chain of observable bubbling to here.